browser/css/normalize.css

```diff
@@ -0,0 +1,461 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
you could have linked to `normalize.css` from a CDN or installed it as a node module. Both are preferable to checking in a library to your source code. Try the CDN linking approach